<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiSync Reader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="hostField">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>17</y>
      <width>94</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelHost">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>17</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="activateRead">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>17</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>68</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="frameValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>79</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>  - 00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="focusValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>79</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>  - 00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="irisValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>79</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>  - 00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="zoomValue">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>79</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>  - 00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerial_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Focus</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerial_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iris</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerial_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
